src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.UnknownUnix.cs

```diff
@@ -82,5 +82,9 @@ private string GetPathToOpenFile()
             throw new PlatformNotSupportedException();
         }
 
+        /// <devdoc>
+        ///    <para>Gets the friendly name of the process.</para>
+        /// </devdoc>
+        public string ProcessName => throw new PlatformNotSupportedException();
```

Comment:
throwing `PlatformNotSupportedException` would be a change in the behavior?

```suggestion
        /// <summary>Gets the friendly name of the process.</summary>
        public string ProcessName
        {
            get
            {
                EnsureState(State.HaveProcessInfo);
                return _processInfo!.ProcessName;
            }
        }
```